src/components/OrderBreakdown/LineItemBasePriceMaybe.js

```diff
@@ -39,7 +39,7 @@ const LineItemBasePriceMaybe = props => {
 
 LineItemBasePriceMaybe.propTypes = {
   lineItems: propTypes.lineItems.isRequired,
-  code: propTypes.lineItemUnitType.isRequired,
+  code: propTypes.lineItemUnitType,
```

Comment:
When using the adoption booking flow, it doesn't have the price, so I only want to use the OrderBreakdown to show the details of the booking: booking start, booking end, and type of service. So if I make it required, it will throw an error.